com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/RenderPass/CustomPass/CustomPass.cs

```diff
@@ -345,9 +465,25 @@ protected void ResolveMSAAColorBuffer(CommandBuffer cmd, HDCamera hdCamera)
             if (!isExecuting)
                 throw new Exception("ResolveMSAAColorBuffer can only be called inside the CustomPass.Execute function");
 
+            // TODO RENDERGRAPH
+            // See how to implement this correctly...
+            // When running with render graph, the design was to have both msaa/non-msaa textures at the same time, which makes a lot of the code simpler.
+            // This pattern here breaks this.
```

Comment:
Yes, we'll need to give both MSAA / non-MSAA color buffers to users so they can handle this stuff on their own.